### ISSUE TITLE
[UI] keyframe display window not updated when moving clips on timeline

### ISSUE THREAD

**Issue by FoxWhiskey** [olive-editor/olive#1636]
**Commit Hash** 4adaedb3

**Platform** Mint 20.1

**Summary**
The keyframe display of the parameter editor is not being updated if a clip is moved backwards or forward on the timeline. An update can be forced by adjusting the width of the keyframe display only.

**Additional Information / Output**
I have attached a simple project file, that demonstrates the problem.
If this a feature rather than a bug: an automatic update would be super helpful to align video keyframes easily to an audio track!!
[update-issue-4adaedb3.ove.zip](https://github.com/olive-editor/olive/files/6496344/update-issue-4adaedb3.ove.zip)

**Comment by FoxWhiskey** [comment]
@ThomasWilshaw : haven't checked in a while, but will do so! Just give me a bit more time for a re-check...

**Comment by FoxWhiskey** [comment]
Well first of all, the observed issue is no longer there, but it has been replaced by an issue which is related in a way:

if a clip is dragged forward or backward in time, the display of keyframes is disabled as if the clip was deselected. If you click on the clip right after the drag, the keyframes appear again.

Nothing serious, admittedly. But usually objects do not  deselect if its properties are changed. It needs a mouse click to a different context, at least... I've attached another sample...
 
[update-issue-6f55bc01.ove.zip](https://github.com/olive-editor/olive/files/7508301/update-issue-6f55bc01.ove.zip)
